### ISSUE TITLE
Profiler not writing to log file on Linux

### ISSUE THREAD

**Issue by mikegwyn17** [DataDog/dd-trace-dotnet#245]
**Describe the bug**
I am trying to run the APM in a Docker Container for a .Net Core API. When I check the log file to see if any events have been recorded it doesn't exist and neither does the /vars/log/datadog directory.

**To Reproduce**
Sample Dockerfile
```
ENV CORECLR_ENABLE_PROFILING 1
ENV CORECLR_PROFILER {846F5F1C-F9AE-4B07-969E-05C26BC060D8}
ENV CORECLR_PROFILER_PATH /opt/datadog/Datadog.Trace.ClrProfiler.Native.so
ENV DD_INTEGRATIONS /opt/datadog/integrations.json
ENV DD_TRACE_LOG_PATH /var/log/datadog/dotnet-profiler.log

RUN curl -LO https://github.com/DataDog/dd-trace-dotnet/releases/download/v0.5.2-beta/datadog-dotnet-apm_0.5.2_amd64.deb \
	&& dpkg -i ./datadog-dotnet-apm_0.5.2_amd64.deb
```
Nuget Packages included in Project:
```
    <PackageReference Include="Datadog.Trace" Version="0.5.2-beta" />
    <PackageReference Include="Datadog.Trace.ClrProfiler.Managed" Version="0.5.2-beta" />
```
Environment Variables after container is run:
```
printenv
CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
ASPNETCORE_URLS=http://*:8080
DD_INTEGRATIONS=/opt/datadog/integrations.json
CORECLR_ENABLE_PROFILING=1
CORECLR_PROFILER_PATH=/opt/datadog/Datadog.Trace.ClrProfiler.Native.so
DD_TRACE_LOG_PATH=/var/log/datadog/dotnet-profiler.log
```

Verification that ClrProfiler installed correctly:
```
opt/datadog# ls
Datadog.Trace.ClrProfiler.Native.so  integrations.json
```

**Runtime environment (please complete the following information):**
- Instrumentation mode: Automatic with Nuget Package
- Tracer version: 0.5.2
- OS: NAME="Debian GNU/Linux", VERSION_ID="9", VERSION="9 (stretch)"
- CLR: .NetCore 2.1

Let me know if you need anymore information from me, the Docker image in question is for a production application so I can't provide the image itself.

**Comment by lucaspimentel** [comment]
@mikegwyn17: On Linux the profiler does not create the folder structure for the log file automatically. This is a known issue. As a workaround, you can create the folder `/vars/log/datadog` yourself or change the `DD_TRACE_LOG_PATH` environment variable.

**Comment by lucaspimentel** [comment]
We consider this issue resolved in [release 1.14.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v1.14.0) where we added a script to the Linux packages that creates the log directories. See #655. Closing this issue.
